src/docfx.website.themes/default/styles/docfx.js

```diff
@@ -710,33 +635,116 @@ function previewPageRefresh() {
-      html += getList(item, cls) || '';
-      html += '</li>';
+  function formList(item, classes) {
+    var level = 1;
+    var model = {
+      items: item
+    };
+    var cls = [].concat(classes).join(" ");
+    return getList(model, cls);
+
+    function getList(model, cls) {
+      if (!model || !model.items) return null;
+      var l = model.items.length;
+      if (l === 0) return null;
+      var html = '<ul class="level' + level + ' ' + (cls || '') + '">';
+      level++;
+      for (var i = 0; i < l; i++) {
+        var item = model.items[i];
+        var href = item.href;
+        var name = item.name;
+        if (!name) continue;
+        html += href ? '<li><a href="' + href + '">' + name + '</a>' : '<li>' + name;
+        html += getList(item, cls) || '';
+        html += '</li>';
+      }
+      html += '</ul>';
+      return html;
     }
-    html += '</ul>';
-    return html;
   }
-}
\ No newline at end of file
+
+  // Show footer
+  (function () {
+    initFooter();
+    $(window).on("scroll", showFooter);
+
+    function initFooter() {
+      if (needFooter()) {
+        shiftUpBottomCss();
+        $("footer").show();
+      } else {
+        resetBottomCss();
+        $("footer").hide();
+      }
+    }
+
+    function showFooter() {
+      if (needFooter()) {
+        shiftUpBottomCss();
+        $("footer").fadeIn();
+      } else {
+        resetBottomCss();
+        $("footer").fadeOut();
+      }
+    }
+
+    function needFooter() {
+      var scrollHeight = $(document).height();
+      var scrollPosition = $(window).height() + $(window).scrollTop();
+      return (scrollHeight - scrollPosition) < 1;
+    }
+
+    function resetBottomCss() {
+      $(".sidetoc").removeClass("shiftup");
+      $(".sideaffix").removeClass("shiftup");
+    }
+
+    function shiftUpBottomCss() {
+      $(".sidetoc").addClass("shiftup");
+      $(".sideaffix").addClass("shiftup");
+    }
+  })();
+
+  // For LOGO SVG
+  // Replace SVG with inline SVG
+  // http://stackoverflow.com/questions/11978995/how-to-change-color-of-svg-image-using-css-jquery-svg-image-replacement
+  jQuery('img.svg').each(function () {
+    var $img = jQuery(this);
+    var imgID = $img.attr('id');
+    var imgClass = $img.attr('class');
+    var imgURL = $img.attr('src');
+
+    jQuery.get(imgURL, function (data) {
+      // Get the SVG tag, ignore the rest
+      var $svg = jQuery(data).find('svg');
+
+      // Add replaced image's ID to the new SVG
+      if (typeof imgID !== 'undefined') {
+        $svg = $svg.attr('id', imgID);
+      }
+      // Add replaced image's classes to the new SVG
+      if (typeof imgClass !== 'undefined') {
+        $svg = $svg.attr('class', imgClass + ' replaced-svg');
+      }
+
+      // Remove any invalid XML tags as per http://validator.w3.org
+      $svg = $svg.removeAttr('xmlns:a');
+
+      // Replace image with new SVG
+      $img.replaceWith($svg);
+
+    }, 'xml');
+  });
+
+  refresh = function(){
```

Comment:
After the change, some of the functions are using declaration and execution:
```
function FuncA(){}
FuncA();
```
style,

Some of the functions are using direct execution:
```
(function(){})();
```
Let's pick one style and keep code consistent.